include/winsparkle.h

```diff
@@ -561,6 +561,7 @@ typedef int(__cdecl* win_sparkle_alternate_appcast_callback_t)(
         - webBrowserUrl : the location to launch in web browser for manual download
         - title : the title of the update
         - description :the description of the update
+        - silent: flag indicating if the the update should be executed silently (by the background comm service) or manually (by the main client app)
 
```

Comment:
Maybe a better wording would be `interactively`, `manually` has a different meaning in this context signified by the `manual` flag.